app/views/suppress_collections/index.html.erb

```diff
@@ -35,7 +35,7 @@
     <tr>
       <td><%= source['name'] %></td>
       <td><%= source['status_updated_by'] %></td>
-      <td><%= source['status_updated_at'].try(:to_datetime).try(:strftime, '%m/%d/%Y %I:%M%p') %></td>
+      <td><%= source['status_updated_at'].try(:to_datetime).try(:strftime, '%d/%m/%Y %I:%M%p') %></td>
```

Comment:
it would be good if you created file: `config/initializers/date_formats.rb` and added your date outputs as constants.

IE `Date::DATE_FORMATS[:dmy] = '%d-%m-%Y'` 
then you can call <date>.to_s(:dmy) instead of having multiple places where your using the strftime formatters.

The same thing goes for all of the other date classes.